Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -73,11 +137,9 @@ iOS WebViews support JavaScript execution by default, so script injection and cr
 
 Besides potential script injection, there's another fundamental WebViews security issue: the WebKit libraries packaged with iOS don't get updated out-of-band like the Safari web browser. Therefore, newly discovered WebKit vulnerabilities remain exploitable until the next full iOS update [#THIEL].
 
-WebViews support different URL schemas, like for example tel. Detection of the [tel:// schema can be disabled](https://developer.apple.com/library/content/featuredarticles/iPhoneURLScheme_Reference/PhoneLinks/PhoneLinks.html "Phone Links on iOS") in the HTML page and will then not be interpreted by the WebView.
-
```

Comment:
why is  this reference removed? Because it is very explanatory

Reply:
It wasn't removed, this one was moved to the test case I created, as it fit's better.